apps/desktop-wallet/src/hooks/useAddressGeneration.tsx

```diff
@@ -52,33 +51,26 @@ interface GenerateOneAddressPerGroupProps {
   skipGroups?: number[]
 }
 
-const addressDiscoveryWorker = new Worker(new URL('../workers/addressDiscovery.ts', import.meta.url), {
-  type: 'module'
-})
-const deriveAddressesInGroupsWorker = new Worker(new URL('../workers/deriveAddressesInGroups.ts', import.meta.url), {
-  type: 'module'
-})
-const deriveAddressesFromIndexesWorker = new Worker(
-  new URL('../workers/deriveAddressesFromIndexes.ts', import.meta.url),
-  { type: 'module' }
-)
+// const addressDiscoveryWorker = new Worker(new URL('../workers/addressDiscovery.ts', import.meta.url), {
+//   type: 'module'
+// })
+// const deriveAddressesInGroupsWorker = new Worker(new URL('../workers/deriveAddressesInGroups.ts', import.meta.url), {
+//   type: 'module'
+// })
+// const deriveAddressesFromIndexesWorker = new Worker(
+//   new URL('../workers/deriveAddressesFromIndexes.ts', import.meta.url),
+//   { type: 'module' }
+// )
```

Comment:
Web workers are disabled because they don't have access to the same `keyring` instance as the rest of the app.

Reply:
FWIW, the web workers were not making things go faster. They were just not blocking the UI thread from rendering.